logpolicy/registry_windows.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package logpolicy
+
+import (
+	"log"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+const (
+	regBase = `SOFTWARE\Tailscale IPN`
```

Comment:
is this constant multiple places now in our tree? I feel like it's spreading.

Maybe move this to two new files in `util/winutil` with build tags for windows and !windows?